pkg/utils/ipnet/ipnet.go

```diff
@@ -18,10 +18,14 @@ var nullString = "null"
 var nullBytes = []byte(nullString)
 var emptyIPNet = net.IPNet{}
 
-// IPNet wraps net.IPNet to get CIDR serialization.
-type IPNet struct {
-	net.IPNet
-}
+type (
+	// IPNet wraps net.IPNet to get CIDR serialization.
+	// +jsonschema noderive
+	// +jsonschema { "type": "string" }
```

Comment:
We do custom marshaling here so the schema needs to be overridden somewhere, this seems like a nice declarative way of doing it.

Reply:
Syntax changed